frontend/src/js/controllers/analyticsCtrl.js

```diff
@@ -0,0 +1,296 @@
+// Invoking IIFE for dashboard
+(function() {
+
+    'use strict';
+
+    angular
+        .module('evalai')
+        .controller('AnalyticsCtrl', DashCtrl);
+
+    DashCtrl.$inject = ['utilities', '$state', '$rootScope'];
+
+    function DashCtrl(utilities, $state, $rootScope) {
+        var vm = this;
+
+        vm.hostTeam = {};
+        vm.teamId = null;
+        vm.currentTeamName = null;
+        vm.challengeListCount = 0;
+        vm.challengeList = {};
+        vm.challengeAnalysisDetail = {};
+        vm.isTeamSelected = false;
+        vm.challengeId = null;
+        vm.currentChallengeDetails = {};
+        vm.currentPhase = [];
+        vm.totalSubmission = [];
+        // vm.challengeCount = 0;
+        // vm.hostTeamCount = 0;
+        // vm.hostTeamExist = false;
+        // vm.participatedTeamCount = 0;
+        // // get token
+
+        // if (vm.teamId == null) {
+        //     vm.isTeamSelected = false;
+        // } else {
+        //     vm.isTeamSelected = true;
+        // }
+        var userKey = utilities.getData('userKey');
+
+        var parameters = {};
+        parameters.url = 'hosts/challenge_host_team/';
+        parameters.method = 'GET';
+        parameters.token = userKey;
+        parameters.callback = {
+            onSuccess: function(response) {
+                var status = response.status;
+                var details = response.data;
+                if (status == 200) {
```

Comment:
It should be an integer. Why it can be a string ....   status === 200;
please guide.